src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/SiteConfiguration.java

```diff
@@ -29,7 +29,19 @@ private void mapPoBoxToSiteId() {
         }
         poBoxToSiteIdMap = getSites()
             .stream()
-            .collect(Collectors.toMap(Sites::getPoBox, Sites::getSiteId));
+            .reduce(
+                    new HashMap<>(),
+                    (m, s) -> {
+                        for (String poBox: s.poBoxes) {
+                            m.put(poBox, s.siteId);
+                        }
+                        return m;
+                    },
+                    (m1, m2) -> {
+                        m1.putAll(m2);
+                        return m1;
+                    }
+            );
```

Comment:
I don't think you need this change. POBox is the key and they are unique. Tested it on #878

Reply:
The approach suggested in #878 is less clean than this one and it is inconsistent with how it is done in processor